app/tests/unit/BurndownTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+use Phragile\Burndown;
+
+class BurndownTest extends TestCase {
+
+	private function mockWithSprint(Sprint $sprint)
+	{
+		return new Burndown(
+			$sprint,
+			$this->getMockBuilder('Phragile\TaskList')->disableOriginalConstructor()->getMock(),
+			$this->getMockBuilder('Phragile\PhabricatorAPI')->disableOriginalConstructor()->getMock());
+	}
+
+	public function dateCountProvider()
+	{
+		return [
+			['2014-01-01', '2014-12-31', 365],
+			['2014-12-01', '2014-12-14', 14],
+			['2014-11-28', '2014-12-2', 5],
```

Comment:
I added a test across the year boundary. The cases with invalid start and end dates are tested here: https://github.com/wmde/phragile/blob/master/app/tests/acceptance/create_sprint.feature#L29 and are caught by the Sprint validate method.